frontend/src/metabase/dashboard/components/DashboardHeader/buttons/ExportAsPdfButton.tsx

```diff
@@ -31,6 +32,7 @@ export const ExportAsPdfButton = (props: ActionIconProps) => {
 
     const cardNodeSelector = `#${DASHBOARD_PDF_EXPORT_ROOT_ID}`;
     return saveDashboardPdf({
+      isEmbeddingSdk,
```

Comment:
As commented elsewhere, we could import this env right directly in the file that defines `saveDashboardPdf` to remove this extra parameter.